resources/views/blog/create.blade.php

```diff
@@ -0,0 +1,64 @@
+<x-layout>
+    <div class="box">
+        <form action="{{ route('articles.store') }}" method="POST">
+            @csrf
+            <h1 class="title is-3" style="text-align: center; margin-top: 0"> Write your article</h1>
+            <div class="icon-text has-text-info">
+                <div class="icon">
+                    <i class="fas fa-info-circle"></i>
+                </div>
+                <p style="text-indent: 0">Please fill out all fields and click 'Submit'</p>
+            </div>
+            <br>
+            {{-- Here are all the form fields --}}
+            <div class="field">
+                <label for="title" class="label">Title</label>
+                <div class="control">
+                    <input type="text" name="title" class="input" autofocus>
+                </div>
+                @error('title')
+                <p class="help is-danger" style="text-indent:0">Title field is required.</p>
```

Comment:
You can also use the error messages generated by Laravel, instead of the fixed text.